### PR TITLE
doc: update changelog with breaking change information from commit footer

### PR DESCRIPTION
**Pull request by Taepper**
### Summary

I unfortunately did not do a good job of adding the commit information from the footer in a correct way, such that it will be displayed in the changelog. I now add this manually and improved to the changelog

## PR Checklist

~- [ ] All necessary documentation has been adapted or there is an issue to do so.~
~- [ ] The implemented feature is covered by an appropriate test.~

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,11 @@
 ### ⚠ BREAKING CHANGES
 
 * generalized wildcard queries ([#458](https://github.com/GenSpectrum/LAPIS-SILO/issues/458))
+  * The preprocessing config field pangoLineageDefinitionFilename has been renamed to lineageDefinitionFilename.
+  * We now accept a YAML lineage definition file instead of a Pango alias key. A script (scripts/alias2lineageDefinitions.py) is provided to transform a list of lineages and an alias into the required file format.
```
Comment (fengelniederhammer) [inline CHANGELOG.md:10]: Do we provide the script?

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,11 @@
 ### ⚠ BREAKING CHANGES
 
 * generalized wildcard queries ([#458](https://github.com/GenSpectrum/LAPIS-SILO/issues/458))
+  * The preprocessing config field pangoLineageDefinitionFilename has been renamed to lineageDefinitionFilename.
```
Comment (fengelniederhammer) [inline CHANGELOG.md:9]:
```suggestion
  * The preprocessing config field `pangoLineageDefinitionFilename` has been renamed to `lineageDefinitionFilename`.
```

File: CHANGELOG.md (hunk 1 above)
Comment (Taepper) [reply CHANGELOG.md:10]: Ah, right. That slipped my attention

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,11 @@
 ### ⚠ BREAKING CHANGES
 
 * generalized wildcard queries ([#458](https://github.com/GenSpectrum/LAPIS-SILO/issues/458))
+  * The preprocessing config field `pangoLineageDefinitionFilename` has been renamed to `lineageDefinitionFilename`.
+  * We now accept a YAML lineage definition file instead of a Pango alias key.
+  * Input and query validation now checks whether the provided lineage exists in the defined lineages, and errors are thrown if validation fails.
+  * The metadata type `pangoLineage` has been removed. `string` with `generateLineageIndex` should be used instead.
```
Comment (fengelniederhammer) [inline CHANGELOG.md:12]:
```suggestion
  * The metadata type `pangoLineage` has been removed. `type: string` with `generateLineageIndex: true` should be used instead.
```

**Review by fengelniederhammer — APPROVED** [review]

**Pull request merged**